src/scroll-timeline-css-parser.js

```diff
@@ -94,6 +100,16 @@ export class StyleParser {
     return null;
   }
 
+  getViewTimelineOptions(timelineName) {
+    // TODO: Take into account the scoping of the ViewTimelines
+    // https://github.com/w3c/csswg-drafts/issues/7047
+    for (let i = this.subjectSelectorToViewTimeline.length - 1; i >= 0; i--)
+      if(this.subjectSelectorToViewTimeline[i].name == timelineName)
+        return this.subjectSelectorToViewTimeline[i];
```

Comment:
target.getParentElement.querySelectorAll(...);  Then pick the last matching value upstream.